tsl/src/continuous_aggs/common.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+#ifndef TIMESCALEDB_TSL_CONTINUOUS_AGGS_COMMON_H
+#define TIMESCALEDB_TSL_CONTINUOUS_AGGS_COMMON_H
+
+#include <postgres.h>
+
+#include <access/reloptions.h>
+#include <access/xact.h>
+#include <catalog/pg_aggregate.h>
+#include <catalog/pg_type.h>
+#include <catalog/toasting.h>
+#include <commands/tablecmds.h>
+#include <commands/tablespace.h>
+#include <miscadmin.h>
+#include <nodes/makefuncs.h>
+#include <nodes/nodeFuncs.h>
+#include <nodes/parsenodes.h>
+#include <nodes/pg_list.h>
+#include <optimizer/optimizer.h>
+#include <parser/parse_func.h>
+#include <parser/parse_oper.h>
+#include <parser/parsetree.h>
+#include <rewrite/rewriteHandler.h>
+#include <rewrite/rewriteManip.h>
+#include <utils/builtins.h>
+#include <utils/syscache.h>
+#include <utils/typcache.h>
+
+#include "errors.h"
+#include "func_cache.h"
+#include "hypertable_cache.h"
+#include "timezones.h"
+#include "ts_catalog/catalog.h"
+#include "ts_catalog/continuous_agg.h"
+
+#define PARTIALFN "partialize_agg"
```

Comment:
super minor, please ignore if it's annoying, but since you're already refactoring the code maybe we can name this `TS_PARTIALFN` instead because now we have two different defs for the same function, once as `PARTIALFN` and once as `TS_PARTIALFN`